src/Microsoft.Identity.Client/Exceptions/CoreErrorMessages.cs

```diff
@@ -165,5 +165,8 @@ public static string CustomWebUiRedirectUriWasNotMatchedToProperUri(string expec
         }
 
         public const string CustomWebUiAuthorizationCodeFailed = "CustomWebUi AcquireAuthorizationCode failed";
+
+        public const string TokenCacheJsonSerializerFailedParse = "MSAL V3 Deserialization failed to parse the cache contents.  Is this possibly an earlier format needed for DeserializeMsalV2?";
```

Comment:
Can we please add in both strings: "(See https://aka.ms/msal-net-3x-cache-breaking-change)"